tests/libcxx/magic_comments.txt

```diff
@@ -5,5 +5,7 @@
 // REQUIRES: c++11
 // REQUIRES: c++11 || c++14
 // REQUIRES: c++11 || c++14 || c++17 || c++20
+// REQUIRES: has-unix-headers
 // REQUIRES: stdlib=libc++
 // UNSUPPORTED: c++14, c++17, c++2a

```

Comment:
Is this line still needed?

Reply:
Yes. This excludes a few tests that require `fork`.

Reply:
I'm going to clear your review and merge this since we've been working to get it in literally for months. Feel free to followup if you see an issue and we'll get it addressed.